daprdocs/content/en/contributing/presentations.md

```diff
@@ -0,0 +1,29 @@
+---
+type: docs
+title: "Giving a presentation on Dapr"
+linkTitle: "Presentations"
+weight: 1500
+description: How to give a presentation on Dapr and examples
+---
+
+We welcome community members giving presentations on Dapr and spreading the word about all the awesome Dapr features! We offer a template PowerPoint file to get started.
+
+{{< button text="Download the Dapr PowerPoint" link="/presentations/Dapr.zip" >}}
```

Comment:
Can we call this "Download the Dapr slide deck"
Also can we call the zip file Dapr-slidedeck.zip to avoid and confusion with Dapr product.

Reply:
Updated.